### ISSUE TITLE
Screen reader does not announce status message which appear after installing the application in Windows.

### ISSUE THREAD

**Issue by msftedad** [wixtoolset/issues#9107]
### WiX Version

5.0.0

### .NET or MSBuild or Visual Studio Version

.NET 8.0.314

### HeatWave Version

1.0.6

### Repro Steps

**User Impact:**
Users who rely on the screen reader cannot determine whether his desired action is performed or not if the change of notification is not notified by AT. It would be much better experienced for users if a confirmation message will appear after any changes made and screen reader reads that message.

**Observation:**
This issue is observed with all three screen readers.

**Pre-Requisite:**

1. Turn on Screen Readers:
Narrator: Win + Shift + Enter
NVDA: Ctrl + Alt + N
JAWS: Ctrl + Alt + J

2. Verbosity:
Narrator: Default
JAWS: Beginner, Highest
NVDA: Default

**Repro steps:**

1. Launch the "NPS Extension for Azure MFA" setup file.
2. "MICROSOFT SOFTWARE LICENSE TERMS" popup window will get open here.
3. Press tab key to move to "I agree to the terms and conditions check box" and activate it using enter key.
4. Press tab key to move to "Install" button and activate using 'Enter' key.
5. Activate Yes button on the "Do you want to make changes to your device pop-up window"
6. Verify whether screen reader is announcing status message which appear after installing the application or not.

### Repro Repo

https://www.microsoft.com/en-us/download/details.aspx?id=54688

### Actual Result

Screen reader does not announce status message which appear after installing the application in Windows.
Narrator remains silent.

**Refer Attachment:**Screen reader is not conveying the Successful installation message to the users.mp4
https://github.com/user-attachments/assets/28bb67bb-8dcb-4efd-bbec-a506c11c8754

### Expected Result

Screen reader should announce status message which appear after installing the application in Windows.

### Acknowledgements

- [x] I acknowledge that this is a fully completed bug report. It is not a question or attempt to get help debugging my issue (because those should be [sent to Discussions](https://github.com/orgs/wixtoolset/discussions/new?category=questions)).
- [x] I am paying the [Open Source Maintenance Fee](https://opensourcemaintenancefee.org/) by [sponsoring the `wixtoolset` project](https://github.com/sponsors/wixtoolset) because I support the maintainers.
